### DISCUSSION TITLE
QExpansionItem label formatting

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#8721] [General - Components / Directives / etc] [answered]
is it possible to format the label on an expansion item? For example, to have some of the words be bold? I have tried several ways without success:
```
:label="<strong>val.name</strong> + ' ' + val.location"
```
and
```
:label="<strong>`${val.name}`</strong> + ' ' + val.location"
```
and many other combos without success. Is this even possible? Thanks!

**Comment by smolinari** [comment, marked as answer]
You can use the header slot.

https://codepen.io/smolinari/pen/YzNpNrB

Scott

**Reply by ssuess** [reply]
Thanks!
